src/rapidata/rapidata_client/order/rapidata_dataset.py

```diff
@@ -66,14 +66,9 @@ def upload_text_datapoint(text_asset: TextAsset | MultiAsset) -> None:
     def add_media_from_paths(
         self,
         media_paths: list[MediaAsset] |  list[MultiAsset], # where multiasset is a list of media assets
-        metadata: list[Metadata] | None = None,
+        metadata: Sequence[Metadata] | None = None,
         max_workers: int = 10,
     ):
-        if metadata is not None and len(metadata) != len(media_paths):
```

Comment:
Why was this removed?

Reply:
wups, good catch. i think i "temporarly" removed it to test something